### PR TITLE
Milestone 3: Events and Operations

### PR DESCRIPTION
**Pull request by AsumVictor**
# Milestone 3: Events and Operations
In this PR, I have events and functionalities to make and display calculations on the screen

 ## Type of change
 * [ ] Created new directory `logic` in the `src` folder
 * [ ] Created `operation.js` and `calculate.js` in the `/logic` folder
 * [ ] Added events in the calculator.js files
 
 # Checklist:
 * [ ]  My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my code
 * [ ] There is no linter error in my code
 * [ ]  My changes generate no new warnings

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
-      <Button value="=" classextend="bg-orange-500" />
+      <Screen total={total} next={next} operation={operation} />
+      <Button handleClick={(e) => makeCalculations(e)} value="AC" />
+      <Button handleClick={(e) => makeCalculations(e)} value="+/-" />
+      <Button handleClick={(e) => makeCalculations(e)} value="%" />
+      <Button handleClick={(e) => makeCalculations(e)} value="÷" classextend="bg-orange-500" />
+      <Button handleClick={(e) => makeCalculations(e)} value="7" />
+      <Button handleClick={(e) => makeCalculations(e)} value="8" />
+      <Button handleClick={(e) => makeCalculations(e)} value="9" />
+      <Button handleClick={(e) => makeCalculations(e)} value="×" classextend="bg-orange-500" />
```
Comment (OyePriscilla) [inline src/components/Calculator.js]: - [ ] I got this error in the process of multiplying two numbers, kindly fix it. 👍 Perhaps you need to replace it with the correct symbol, which is the standard multiplication operator in JavaScript. 👍 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @AsumVictor 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Calculator app handled state with useState ✔️
Click events are implemented for all buttons ✔️
App worked as expected ❎ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsumVictor** [comment]
Okay thank you so much.

File: src/components/Calculator.js (hunk 1 above)
Comment (AsumVictor) [reply src/components/Calculator.js]: Thank you, I have solved it.

**Review by Ismailco — APPROVED** [review]
Thank you very much!

**Comment by Ismailco** [comment]
@AsumVictor, **Please don't edit comments**

**Pull request merged**